### PR TITLE
fixup!: backslash is removed from sd and tn-ro

### PR DESCRIPTION
**Pull request by Kate-Lyndegaard**
ING-1589

### CODE REVIEW

**Review by stempler — COMMENTED** [review]
How are these xmi files created? Would there be the option to drop the now duplicate namespace prefix `tn-ro_4.0_1`?

What about these places?

- https://github.com/wetransform-os/portrayal-repository/blob/829b0851dce6f3846ba52f2e8b442034a19a7d7d/config.style#L1107
- https://github.com/wetransform-os/portrayal-repository/blob/829b0851dce6f3846ba52f2e8b442034a19a7d7d/config.style#L1116
- https://github.com/wetransform-os/portrayal-repository/blob/829b0851dce6f3846ba52f2e8b442034a19a7d7d/config.style#L1194
- https://github.com/wetransform-os/portrayal-repository/blob/829b0851dce6f3846ba52f2e8b442034a19a7d7d/config.style#L1201

**Comment by Kate-Lyndegaard** [comment]
@stempler I believe these files are created dynamically with Extend. I don't know if `tn-ro_4.0_1` can be deleted. I will remove the trailing backslashes from the other files you identified.

**Comment by stempler** [comment]
@Kate-Lyndegaard Don't you need to run the tool again to have a consistent state?
There are more occurrences also in https://github.com/wetransform-os/portrayal-repository/blob/master/generated/generated.style

**Comment by Kate-Lyndegaard** [comment]
> Don't you need to run the tool again to have a consistent state?

I cannot currently run Xtext. I will look into it with Florian.

**Comment by Kate-Lyndegaard** [comment]
@stempler Unfortunately we could not get the Xtext/Xtend to run the generation. I manually updated the values.

File: src-gen/report.html [hunk 1]
```diff
@@ -6,15 +6,15 @@
 		    border-collapse: collapse;
 		    width: 100%;
 		}
-
```
Comment (JohannaOtt) [inline src-gen/report.html:9]: Are the format changes in the report.html file commited intentionally?

**Review by stempler — APPROVED** [review]

File: src-gen/report.html (hunk 1 above)
Comment (Kate-Lyndegaard) [reply src-gen/report.html:9]: @JohannaOtt The only changes I made to this file were updates on the related namespace references.

Comment (Kate-Lyndegaard) [reply src-gen/report.html:9]: @stempler I don't know how these changes were introduced. I am using ATOM.

Comment (Kate-Lyndegaard) [reply src-gen/report.html:9]: @JohannaOtt We discussed this, and as it is a generated file we cannot generate anyways, we can leave it as it is.

**Pull request merged**